content/en/blog/_posts/2019-06-25-recap-of-contributor-summit-bcn-2019.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: blog
+title: 'Recap of Kubernetes Contributor Summit Barcelona 2019'
+date: 2019-06-25
+---
+
+**Author**: Jonas Rosland (VMware)
+
+First of all, **THANK YOU** to everyone who made the Kubernetes Contributor Summit in Barcelona possible. We had an amazing team of volunteers tasked with planning and executing the event, and it was so much fun meeting and talking to all new and current contributors during the main event and the pre-event celebration.
+
+Contributor Summit in Barcelona kicked off KubeCon + CloudNativeCon in a big way as it was the **largest contributor summit** to date with 331 people signed up, and only 9 didn't pick up their badges!
+
+## Contributor Celebration
+
+Sunday evening before the main event we held a **Contributor Celebration**, which was very well attended. We hope that all new and current contributors felt welcome and enjoyed the food, the music, and the company.
+
+![contributor-celebration2](https://live.staticflickr.com/65535/46981485515_561bb324b2_z.jpg)
+![contributor-celebration](https://live.staticflickr.com/65535/46981484655_8122564557_z.jpg)
+
+## New Contributor Workshops
```

Comment:
This wouldn't be a bad place to cross link the playlist on youtube so people who didn't attend can check the content out: https://www.youtube.com/watch?list=PL69nYSiGNLP2WTJ6P8sQenhf0RY-JqF5L&v=BQ7y2TFOzF4